package.json

```diff
@@ -10,6 +10,7 @@
   "dependencies": {
     "@babel/polyfill": "^7.4.4",
     "@sentry/browser": "^4.0.6",
+    "@typeform/embed": "^0.13.1",
```

Comment:
We removed the Typeform integration from our app a little while ago. Did you perhaps have a merge conflict in which you accidentally added this back in? Could you remove this dependency?